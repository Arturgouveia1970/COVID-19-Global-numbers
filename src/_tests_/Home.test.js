import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Home from '../components/Home';

describe('snapshots testing', () => {
  test('snapshots for Home component', () => {
    const rendererComponents = renderer
      .create(
        <Provider store={store}>
          <Home />
        </Provider>,
      )
      .toJSON();
    expect(rendererComponents).toMatchSnapshot();
  });
});
