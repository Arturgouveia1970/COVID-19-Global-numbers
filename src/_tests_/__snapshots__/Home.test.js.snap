// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots testing snapshots for Home component 1`] = `null`;
